src/Components/ReplyForm/ReplyForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './ReplyForm.scss';
+import sendIcon from '../../assets/send.svg';
+
+interface IProps {
+  addReply: (newPost: string) => void;
+}
+
+interface IReply {
+  reply: string;
+  disabled: boolean;
+}
```

Comment:
Once we're done with components let's look through and figure out where the best place is for all the interfaces